kratos/python_scripts/run_tests.py

```diff
@@ -42,7 +40,7 @@ def GetModulePath(module):
 
     '''
 
-    return imp.find_module(module)[1]
+    return os.path.dirname(KtsMp.__file__)
```

Comment:
This is no longer general I'd say => `GetModulePath` should now be named `GetKratosMultiphysicsPath`